### DISCUSSION TITLE
Is it possible to switch SkiaSharp to pure C# implementation in the future?

### DISCUSSION THREAD

**Discussion by sgf** [mono/SkiaSharp#2906] [General]
I don't think I need to say much about the benefits of pure C# implementation.

**Comment by mattleibow** [comment]
Not really, we are a wrapper. The amount of work and time it will take to write a massive library like skia is just not possible. Google has multiple teams and numerous years of effort in this. 

If you want pure c# then ImageSharp is your friend.

**Reply by sgf** [reply]
OK, thank you very much for your answer.
